README.md

```diff
@@ -46,14 +46,14 @@ A `tests.toml` file has this format:
 ```toml
 # This is an auto-generated file.
 #
-# Regenerating this file will:
-# - Update the `description` property
-# - Update the `reimplements` property
-# - Remove `include = true` properties
-# - Preserve any other properties
+# Regenerating this file via `configlet sync` will:
+# - Recreate every `description` key/value pair
+# - Recreate every `reimplements` key/value pair, where they exist in problem-specifications
+# - Remove any `include = true` (an omitted `include` key implies inclusion)
+# - Preserve any other key/value pair
 #
-# As regular comments will be removed when this file is regenerated, comments
-# can be added in a "comment" key
+# As other TOML comments (using the # character) will be removed when this file
```

Comment:
I found the "other" a bit weird. I know what you're trying to describe here, but I'm not sure it is very clear with this wording. The question then becomes: how _do_ we want to word it? Maybe `user-added` comments? Or `Any comments (except these introductory comments)`?

Reply:
Yeah, "other" was bad. I've added a commit for "user-added".